INSTALL

```diff
@@ -26,3 +26,15 @@ Brief Instructions
    with unixODBC 2.2.11.
 
    For any other information please see the Users Guide.
+
+Docker installation (for Debian-based images)
+
+    RUN apt install -y git automake autoconf libtool make gcc perl gettext unixodbc-dev
```

Comment:
Not sure that you need `gettext`, can you check?
Would not be better to download a snapshot and avoid `libtool`, `automake`, `autoconf` and `perl`?